src/main/java/com/fvv/bookstore/controller/CustomerControllerImpl.java

```diff
@@ -118,4 +120,18 @@ private void validateCustomer(final Customer customer) throws PersonValidationEx
 			throw new PersonValidationException(sb.toString());
 		}
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public List<Customer> listCustomersByName(final String name) throws PersonNotFoundException, ControllerException {
+		try {
+			List<Customer> customers = this.customerDao.listCustomersByName(name);
+			Collections.sort(customers, new CustomerSort());
```

Comment:
in java8 you can simplify this by doing:
```
customers.sort(new CustomerSort());
```

Reply:
ok